scripts/bench/compare_results.py

```
@@ -0,0 +1,149 @@
+    category_deltas = {}
+    for cat_id in sorted(set(list(b_cats.keys()) + list(t_cats.keys()))):
+        cat_key = str(cat_id)
+        b_cat = b_cats.get(cat_key, {})
+        t_cat = t_cats.get(cat_key, {})
+
+        b_cat_acc = b_cat.get("accuracy", 0)
+        t_cat_acc = t_cat.get("accuracy", 0)
+        cat_delta = t_cat_acc - b_cat_acc
+
+        name = category_names.get(int(cat_id), f"Category {cat_id}")
+        b_detail = f"{b_cat.get('correct', '?')}/{b_cat.get('total', '?')}"
```

Ruff (0.15.1)

[error] Local variable `b_detail` is assigned to but never used

Remove assignment to unused variable `b_detail`

(F841)